test/unit/rubygem_test.rb

```diff
@@ -496,7 +496,7 @@ class RubygemTest < ActiveSupport::TestCase
     end
 
     should "only latest downloaded versions" do
-      assert_equal [@thin, @rake, @json, @thor, @rack],        Rubygem.downloaded
+      assert_equal [@thin, @rake, @json, @thor, @rack],        Rubygem.downloaded(5)
```

Comment:
@sferik I'm not sure but I think we can leave off tests parameters as 5 for easiness.